### ISSUE TITLE
res.send is not a function

### ISSUE THREAD

**Issue by markmcdermid** [davezuko/react-redux-starter-kit#483]
```js
 TypeError: res.send is not a function
      at processRequest (/Users/markmcdermid/Desktop/test/node_modules/webpack-dev-middleware/middleware.js:189:8)
      at ready (/Users/markmcdermid/Desktop/test/node_modules/webpack-dev-middleware/middleware.js:96:20)
      at webpackDevMiddleware (/Users/markmcdermid/Desktop/test/node_modules/webpack-dev-middleware/middleware.js:162:3)
      at apply-express-middleware.js:10:5
      at new Promise (/Users/markmcdermid/Desktop/test/node_modules/core-js/library/modules/es6.promise.js:197:7)
      at applyExpressMiddleware (apply-express-middleware.js:5:10)
      at _callee$ (webpack-dev.js:23:25)
      at tryCatch (/Users/markmcdermid/Desktop/test/node_modules/babel-runtime/regenerator/runtime.js:88:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/markmcdermid/Desktop/test/node_modules/babel-runtime/regenerator/runtime.js:341:22)
      at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/markmcdermid/Desktop/test/node_modules/babel-runtime/regenerator/runtime.js:121:21)
      at step (/Users/markmcdermid/Desktop/test/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
      at /Users/markmcdermid/Desktop/test/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:7
      at new Promise (/Users/markmcdermid/Desktop/test/node_modules/core-js/library/modules/es6.promise.js:197:7)
      at /Users/markmcdermid/Desktop/test/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12
      at koaWebpackDevMiddleware (webpack-dev.js:22:25)
      at dispatch (/Users/markmcdermid/Desktop/test/node_modules/koa-compose/index.js:41:32)
```

After cloning I get this after installing and running on MAC OS, npm@3.4.0 and node@4.2.2 without changing anything.

Note that it was working quite recently (I only downloaded it for the first time 4 days ago and I've been building fine with it). Only re-cloned today and now nothing will run.

**Comment by gbelken** [comment]
I am able to reproduce this exact same issue on Both Windows and Linux.  I had originally cloned this repo a few days earlier on a mac and that worked fine until I uninstalled all npm packages and re-ran npm install, I suspect this issue might be related to a dependency?

**Comment by davezuko** [comment]
@gbelken that's my thought, a bit backed up w/ work this week but I'll try to dig into this ASAP.

**Comment by gbelken** [comment]
@davezuko I will try and see what I can dig up, If I can find anything I will report back.  Thanks!

**Comment by davezuko** [comment]
thank you, it's much appreciated.

**Comment by gbelken** [comment]
@davezuko, Looks like the dependency that caused the issue is: webpack-dev-middleware.  Looks like this commit is the culprit from 10 days ago and must have made it into the 1.5.0 release.  By pinning back to 1.4.0 everything works again.

**Comment by davezuko** [comment]
@gbelken thanks so much for figuring this out so quickly, I'm sure this will save a lot of other users from unneeded headaches (looking forward to our apps breaking tomorrow at work!).

Resolved w/ https://github.com/davezuko/react-redux-starter-kit/commit/215d1d76b00baa0808444952e95c4891ef60ade8

**Comment by gbelken** [comment]
@davezuko glad to help!

**Comment by markmcdermid** [comment]
Thanks for the fix :)

**Comment by hzoo** [comment]
https://github.com/webpack/webpack-dev-middleware/pull/59 fixes it in 1.5.1
